integrations/docker/images/chip-build/version

```diff
@@ -1 +1 @@
-0.5.78 Version bump reason: [Tizen] Fix environment variable order, add missing tizen dependencies
+0.5.79 Version bump reason: [Ameba] Add reset-watermarks and read thread-metrics function
```

Comment:
why is a dockerfile change required for this ?

I thought these items are implemented in chip source code.

Reply:
This update meant to adapt the 'new functions added in chip source code' in the platform sdk . The description is not accurate, I will change it. Thanks